src/util/path.js

```diff
@@ -811,6 +811,30 @@
     });
   }
 
+  /**
+   * Returns an array of path commands to create a regular polygon
+   * @param {number} radius
+   * @param {number} numVertexes
+   * @returns {(string|number)[][]} An array of SVG path commands
+   */
+  function getRegularPolygonPath(radius, numVertexes) {
```

Comment:
can we swap and have numVertexes first? radius can be defaulted to something when we have es6

Reply:
99d5183e